nodejs-jquery/nodejs-jquery.spec

```diff
@@ -0,0 +1,57 @@
+%{?scl:%scl_package nodejs-%{npm_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global npm_name jquery
+
+Name: %{?scl_prefix}nodejs-%{npm_name}
+Version: 1.11.3
+Release: 1%{?dist}
+Summary: JavaScript library for DOM operations
+License: MIT
+Group: Development/Libraries
+URL: https://github.com/jquery/jquery.git
+Source0: http://registry.npmjs.org/jquery/-/jquery-1.11.3.tgz
+BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
+Requires: nodejs(engine)
```

Comment:
The package name is using SCL macros, but these aren't - did you want to include %{?scl_nodejs} here so they switch to the SCL collection when present (tfm can be rebuilt with it)?  Otherwise I'm not sure the SCL prefixes are necessary.